profile.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+// iron-node does not work with forked processes
+// This cli command will run a single file in the current process.
+// Intended to be used with iron-node for profiling purposes.
+
+var path = require('path');
+var meow = require('meow');
+var Promise = require('bluebird');
+var pkgConf = require('pkg-conf');
+var arrify = require('arrify');
+var findCacheDir = require('find-cache-dir');
+var uniqueTempDir = require('unique-temp-dir');
+var EventEmitter = require('events').EventEmitter;
+var CachingPrecompiler = require('./lib/caching-precompiler');
+var globals = require('./lib/globals');
+
+// copy original console and console.log so tests can't manipulate them.
+var originalConsole = console;
+var consoleLog = originalConsole.log;
```

Comment:
Yeah, I know. Just thought it would be cleaner to keep all overridden globals in our `globals` file.